README.md

```diff
@@ -1,18 +1,20 @@
 ## How to Use This Stack
 
 1. Install VirtualBox
- - I've successfully used 4.2.22, 4.3.8, 4.3.12, 4.3.18
+ - I've successfully used 4.2.22, 4.3.8, 4.3.12, 4.3.18, 6.0.8
 2. Install VirtualBox Extension Pack
  - This is necessary to be able to PXE boot a VM without a iso file or
    using the default Intel network adapter
  - I had sucess on VBox 4.2.22 using the PCnet-FAST III (Am79C973)
    adapter without the extension pack but others reported issues to me
 2. Install Vagrant
-  - I've personally used 1.4.3, 1.6.3 in the past and 1.6.5 right now.
+  - I've personally used 1.4.3, 1.6.3 in the past and 1.6.5 right now. Version
+2.2.4 also works.
 3. `vagrant plugin install oscar`
-  - Optional: `vagrant plugin install vagrant-multiprovider-snap`
+  - Optional: `vagrant plugin install vagrant-multiprovider-snap`; this is not
+needed for version 6.x.
 4. `vagrant up`
-  - Optional: `vagrant snap take`
+  - Optional: `vagrant snap take`; this is not needed for version 6.x.
```

Comment:
It sounds like it was optional before? What makes it not needed here?

Reply:
Agreed; taking a snapshot is a bit of a time-save for future uses. I don't think this has changed with 6.x.